hadoop-ozone/dist/src/main/compose/ozone/docker-config

```diff
@@ -25,6 +25,7 @@ OZONE-SITE.XML_ozone.scm.client.address=scm
 OZONE-SITE.XML_ozone.replication=3
 OZONE-SITE.XML_hdds.datanode.dir=/data/hdds
 OZONE-SITE.XML_hdds.profiler.endpoint.enabled=true
+OZONE-SITE.XML_hdds.scm.safemode.min.datanode=3
```

Comment:
This will make it impossible to use these environments with a single datanode without modifying the config locally.

I would like to propose an alternative solution:

1. define this config in the `environment` section in `docker-compose.yaml` using a variable that defaults to 1:
   `- "OZONE-SITE.XML_hdds.scm.safemode.min.datanode=${SAFEMODE_MIN_DATANODES:-1}"`
2. set the variable to 3 in `testlib.sh`:
   `export SAFEMODE_MIN_DATANODES=3`

Reply:
Yes, this is the problem what I tried to describe in #238 I am fine with the suggested approach but it makes more complex the definition. 

What I am thinking is to create a `simple` compose definition which can work with one datanode (and we can adjust there the replication factor and the s3 storage type as well).

Almost all the tested functionality requires datanode=3, it seems to be enough to have one cluster which can work with one datanode...

Reply:
OK, I'm fine with the hard-coded values in order to get acceptance tests in a good shape.  We can refine it later.  Until then, config can be edited locally if needed.

Reply:
During an offline conversation I understood your use case: at some cases it can be useful to make the compose folder usable just with one datanode (eg. when ui / recon or shell scripts is tested).

I pushed a new commit to experiment with your proposal.